### DISCUSSION TITLE
Ready to start programming

### DISCUSSION THREAD

**Discussion by BobbyPr** [missionpinball/missionpinball#113] [Q&A]
Attempting to recreate 8 Ball by Bally table. 
Using Cobrapin, I've now managed to hook up all my switches and coils and have a basic game successfully running i.e after step 15 of the MPF tutorial and I'm now ready to start programming for all the complexities of the scoring. I've been reading up on this but it all seems so overwhelming . Where to start? How to start? Any tips would be appreciated. 
Thanks!

**Comment by toomanybrians** [comment]
One thing that may help is to search GitHub for MPF repos you can look at for ideas and examples? One way to do this is to search for YAML files with `config_version=5` in them, like [this](https://github.com/search?q=config_version%3D5+language%3Ayaml&type=code).

**Comment by BobbyPr** [comment]
i’m not sure any of those apply. i was just wondering how to attack this or break it down into manageable parts. i thought just doing one target at a time but that doesn’t seem to work.i’m trying to do an existing simple game (8 ball) but just don’t know where to startSent from my iPhoneOn Oct 2, 2023, at 7:11 PM, Brian Madden ***@***.***> wrote:﻿
One thing that may help is to search GitHub for MPF repos you can look at for ideas and examples? One way to do this is to search for YAML files with config_version=5 in them, like this.
